.circleci/config.yml

```diff
@@ -95,6 +93,40 @@ jobs:
       xcode: "9.0"
     working_directory: ~/repo
     steps:
+      - restore_cache:
+          keys:
+            - brew-cache-2
+      - run: brew install socat docker-machine-driver-xhyve docker &&
+             sudo chown root:wheel /usr/local/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve &&
+             sudo chmod u+s /usr/local/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+      - run:
+          name: Docker Virtual Machine
+          command: docker-machine create default --driver xhyve && sleep 7200
+          background: true
+      - run: until docker-machine env; do sleep 3; done && docker-machine env >> ${BASH_ENV}
+      - run: docker system info
+      - save_cache:
+          key: brew-cache-2
+          paths:
+            - /usr/local/Homebrew
+            - ".docker"
+          when: always
+      - run:
+          name: Registry Port Forward (IPv4)
+          command: socat tcp4-listen:5000,reuseaddr,fork tcp:$(docker-machine ip):5000
+          background: true
+      - run:
+          name: K3s Port Forward (IPv4)
+          command: socat tcp4-listen:6443,reuseaddr,fork tcp:$(docker-machine ip):6443
+          background: true
+      - run:
+          name: Registry Port Forward (IPv6)
+          command: socat tcp6-listen:5000,reuseaddr,fork tcp:$(docker-machine ip):5000
+          background: true
+      - run:
+          name: K3s Port Forward (IPv6)
+          command: socat tcp6-listen:6443,reuseaddr,fork tcp:$(docker-machine ip):6443
+          background: true
```

Comment:
This is a big enough chunk that maybe it should e moved in to k3sctl or something, for every other repo making use of dtest.

Reply:
Totally agree, but I'd prefer to release first and address that later. I feel like this sojurn has already bundled up too much work. ;-)